library/mono

```diff
@@ -83,13 +83,23 @@
 4.6.0-onbuild: git://github.com/mono/docker@9fdd0e79b4eb3e7e7e818fbd58bd324d4c5ab7e1 4.6.0.245/onbuild
 
 4.6.1.3: git://github.com/mono/docker@f0f1e0b9f693bd7c58d9623d402c8dc69234bbe1 4.6.1.3
```

Comment:
Ah now I understand what you mean. I'm not 100% sure what our policy is on that, but I could imagine we'd just issue a new 4.6.x release so even 4.6.0.245 and 4.6.1.5 wouldn't get a fix.

Can we wait until such a scenario actually happens and remove the versions at that point? Otherwise I'd need to talk to a few people on Monday.